repository/src/main/java/org/apache/atlas/repository/audit/ESBasedAuditRepository.java

```diff
@@ -258,6 +292,15 @@ private EntityAuditSearchResult getResultFromResponse(String responseString) thr
         return searchResult;
     }
 
+    private AtlasEntityHeader fetchAtlasEntityHeader(String domainGUID) throws AtlasBaseException {
+        try {
+            AtlasEntityHeader entityHeader = entityGraphRetriever.toAtlasEntityHeader(domainGUID);
+            return entityHeader;
+        } catch (AtlasBaseException e) {
+            throw new AtlasBaseException(e);
+        }
+    }
```

Comment:
Nit pick - is this extra method helping? If there is only a single line, it is extra hop for code reader to follow.